### ISSUE TITLE
WinForms - WebView2 App Crash when targeting .NET5 ARM64 due to System.Windows.Rect

### ISSUE THREAD

**Issue by snickler** [MicrosoftEdge/WebViewFeedback#235]
```
Application: WindowsFormsApp2.exe
CoreCLR Version: 5.0.20.27907
.NET Version: 5.0.0-preview.6.20279.7
Description: The process was terminated due to an unhandled exception.
Exception Info: System.TypeLoadException: Could not load type 'System.Windows.Rect' from assembly 'WindowsBase, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'.
   at Microsoft.Web.WebView2.WinForms.WebView2.InitCoreWebView2Async(CoreWebView2Environment environment)
   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine)
   at Microsoft.Web.WebView2.WinForms.WebView2.InitCoreWebView2Async(CoreWebView2Environment environment)
   at Microsoft.Web.WebView2.WinForms.WebView2.EnsureCoreWebView2Async(CoreWebView2Environment environment)
   at Microsoft.Web.WebView2.WinForms.WebView2.set_Source(Uri value)
   at WindowsFormsApp2.Form1.InitializeComponent() in C:\Users\jerem\Downloads\WindowsFormsApp2\WindowsFormsApp2\Form1.Designer.cs:line 49
   at WindowsFormsApp2.Form1..ctor() in C:\Users\jerem\Downloads\WindowsFormsApp2\WindowsFormsApp2\Form1.cs:line 19
   at WindowsFormsApp2.Program.Main() in C:\Users\jerem\Downloads\WindowsFormsApp2\WindowsFormsApp2\Program.cs:line 20

```

When instantiating a new instance of `Microsoft.Web.WebView2.WinFoms.WebView2` in a WinForms project on ARM64, setting the Size or Source properties will result in a crash due to the usage of `System.Windows.Rect`. Digging into the decompilation of the WinForms WebView2 DLL, the culprit is the `CoreWebView2Controller.Bounds` property. As WPF support support isn't ready for ARM64 in the .NET 5, it seems that relying on a WPF type for use in a WinForms project presents an issue. 

Is there any way that the `Bounds` property can be abstracted from explicitly being tied to `Rect`? 

Thanks!

**Comment by pagoe-msft** [comment]
@snickler thanks for raising this issue, I've filed a bug in our backlog.

**Comment by RussKie** [comment]
Windows Forms stack shouldn't use WPF primitives, instead use primitives provided by System.Drawing, e.g. https://docs.microsoft.com/en-us/dotnet/api/system.drawing.rectangle?view=netcore-3.1

**Comment by oggy22** [comment]
@RussKie that is correct. This is actually our wrapper DLL used both by wpf and winforms. We will switch to

**Comment by oggy22** [comment]
.... System.Drawing

**Comment by champnic** [comment]
This has been resolved and should be available in our next SDK release :)

**Comment by champnic** [comment]
The fix is now available in the latest prerelease package, 0.9.628-prerelease. Thanks!
